config/manager/manager.yaml

```diff
@@ -24,7 +24,7 @@ spec:
         control-plane: controller-manager
     spec:
       hostNetwork: true
-      serviceAccountName: windows-machine-config-operator
```

Comment:
Are you certain there are no other side effects? I suggest a separate PR for this.

Reply:
There should be no issue as it is a duplicate field in the yaml (`serviceAccountName` defined at line 47 already).

Reply:
This sort of fixe should go into a separate PR and not when introducing a new change. Consider if we need to revert this PR for some reason; that revert would including removing this useful change.

Reply:
That makes sense, thanks. New PR opened [here](https://github.com/openshift/windows-machine-config-operator/pull/464)